app/src/main/java/org/simple/clinic/scheduleappointment/patientFacilityTransfer/PatientFacilityChangeActivity.kt

```diff
@@ -16,14 +19,27 @@ class PatientFacilityChangeActivity : AppCompatActivity() {
   @Inject
   lateinit var locale: Locale
   
+  private lateinit var component: PatientFacilityChangeComponent
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.screen_patient_facility_change)
   }
 
   override fun attachBaseContext(baseContext: Context) {
-    val contextWithOverridenLocale = LocaleOverrideContextWrapper.wrap(baseContext, locale)
-    super.attachBaseContext(ViewPumpContextWrapper.wrap(contextWithOverridenLocale))
+    component = ClinicApp
+        .appComponent
+        .patientFacilityChangeComponentBuilder()
+        .activity(this)
+        .build()
+    component.inject(this)
+
+    val wrappedContext = baseContext
+        .wrap { LocaleOverrideContextWrapper.wrap(it, locale) }
+        .wrap { InjectorProviderContextWrapper.wrap(it, component) }
```

Comment:
This context wrapper is only necessary if there are nested screens/views within `PatientFacilityChangeActivity` that need to be field-injected. Since this activity does not do this at the moment, we can remove this.

Reply:
Removed